src/thunderbots/software/test/util/ros_messages.cpp

```diff
@@ -1,7 +1,24 @@
 #include "util/ros_messages.h"
 #include <gtest/gtest.h>
 
-TEST(ROSMessageUtilTest, create_ball_from_ros_message)
+using namespace std::chrono;
+
+class ROSMessageUtilTest : public ::testing::Test
+{
+   protected:
+    void SetUp() override
+    {
+        auto epoch = time_point<std::chrono::steady_clock>();
+        // An arbitrary fixed point in time. 10000 seconds after the epoch
```

Comment:
Maybe add a quick note on why we have this here (ie. to make the tests deterministic).

Reply:
done